pkgs/applications/networking/irc/epic5/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, fetchurl, pkgs, openssl
+  , ncurses, libiconv, tcl }:
+
+stdenv.mkDerivation rec {
+  name = "epic5-${version}";
+  version = "2.0.1";
+
+  src = fetchurl {
+    url = "http://ftp.epicsol.org/pub/epic/EPIC5-PRODUCTION/${name}.tar.xz";
+    sha256 = "1ap73d5f4vccxjaaq249zh981z85106vvqmxfm4plvy76b40y9jm";
+    };
+
+   # Darwin needs libiconv, tcl; while Linux build don't
+ 
+   buildInputs = [ openssl ncurses ]  
+   ++ stdenv.lib.optional 
```

Comment:
While it works, it's preferred not to have nested lists in buildInputs. You can use `stdenv.lib.optionals` to add a list of conditional inputs.